### DISCUSSION TITLE
List issues > create branch from issue?

### DISCUSSION THREAD

**Discussion by airtonix** [pwntester/octo.nvim#392] [Q&A] [answered]
Has anyone come up with a solution to this workflow ? 

- list issues
- pick and issue
- `????`
- have branch created from issue with a certain branch name format.

**Comment by pwntester** [comment]
Interesting idea, it could be implemented by adding an action like opening in browser [here](https://github.com/pwntester/octo.nvim/blob/master/lua/octo/pickers/telescope/provider.lua#L172)

**Comment by arnold-c** [comment]
I haven't in Octo, but the GH CLI has an option to do this using the command `gh issue develop #X --checkout` (https://cli.github.com/manual/gh_issue_develop), which I have mapped to a git alias `idev = "!gh issue develop \"$1\" --checkout #"`, so opening up a terminal (usually a quick floatterm) and entering the command `git idev #X` works. There is some friction in this approach (needing to open up a terminal rather than staying in NeoVim, and needing to find the correct issue number), but it's faster than the alternatives I've come across

**Comment by LeonardoMor** [comment]
Well, you can use octo to see the issue number and use `gh`.

But still, it'd be nice to create a branch based on an issue and checkout to it from octo.

**Comment by williambdean** [comment, marked as answer]
Edit: try out the `:Octo issue develop` command first develop in #609
